src/styles/blocks/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  box-sizing: border-box;
```

Comment:
move box-sizing to * selector

Reply:
https://github.com/mate-academy/layout_product-cards/pull/3400/files#r1272520501
didn't fix it, and delete box-sizing in all other places